From 81fcb24195160c63d293d8f6ecec0eda4bcef61e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonah=20Br=C3=BCchert?= <jbb.prv@gmx.de>
Date: Thu, 28 Mar 2019 19:48:04 +0100
Subject: [PATCH] Finish renaming of ukranian translation

---
 OpenBoard.pro | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/OpenBoard.pro b/OpenBoard.pro
index 87d0eb9d..3f080397 100644
--- a/OpenBoard.pro
+++ b/OpenBoard.pro
@@ -383,10 +383,10 @@ macx {
        TRANSLATION_gl.path = "$$RESOURCES_DIR/gl.lproj"
        QMAKE_BUNDLE_DATA += TRANSLATION_gl
    }
-   exists(resources/i18n/OpenBoard_ua.qm) {
-       TRANSLATION_ua.files = resources/i18n/OpenBoard_ua.qm \
+   exists(resources/i18n/OpenBoard_uk.qm) {
+       TRANSLATION_ua.files = resources/i18n/OpenBoard_uk.qm \
            resources/i18n/localizable.strings
-       TRANSLATION_ua.path = "$$RESOURCES_DIR/ua.lproj"
+       TRANSLATION_ua.path = "$$RESOURCES_DIR/uk.lproj"
        QMAKE_BUNDLE_DATA += TRANSLATION_ua
    }
    exists(resources/i18n/OpenBoard_hu.qm) {
@@ -470,7 +470,7 @@ TRANSLATIONS = resources/i18n/OpenBoard_en.ts \
    resources/i18n/OpenBoard_tr.ts \
    resources/i18n/OpenBoard_cs.ts \
    resources/i18n/OpenBoard_gl.ts \
-   resources/i18n/OpenBoard_ua.ts \
+   resources/i18n/OpenBoard_uk.ts \
    resources/i18n/OpenBoard_hu.ts \
    resources/i18n/OpenBoard_mg.ts
 
-- 
2.20.1

