From e446e28be8e69f2b0c2e7fd65b509ad8bf9d0a2d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonah=20Br=C3=BCchert?= <jbb.prv@gmx.de>
Date: Tue, 11 Dec 2018 18:45:20 +0100
Subject: [PATCH 4/9] Use QStandardPaths to locate resources

---
 src/frameworks/UBPlatformUtils_linux.cpp | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/src/frameworks/UBPlatformUtils_linux.cpp b/src/frameworks/UBPlatformUtils_linux.cpp
index e80dff30..4d620355 100644
--- a/src/frameworks/UBPlatformUtils_linux.cpp
+++ b/src/frameworks/UBPlatformUtils_linux.cpp
@@ -34,6 +34,7 @@
 #include <QDBusConnectionInterface>
 #include <QDBusInterface>
 #include <QDBusMetaType>
+#include <QStandardPaths>
 
 #include <unistd.h>
 #include <X11/keysym.h>
@@ -53,7 +54,13 @@ void UBPlatformUtils::init()
 
 QString UBPlatformUtils::applicationResourcesDirectory()
 {
-    return QApplication::applicationDirPath();
+    QString appDataPath = QStandardPaths::locate(QStandardPaths::GenericDataLocation,
+                                                 QApplication::applicationName() + "/library/applications/",
+                                                 QStandardPaths::LocateDirectory);
+    if (appDataPath.isEmpty())
+        return QApplication::applicationDirPath();
+    else
+        return appDataPath + "../../";
 }
 
 void UBPlatformUtils::hideFile(const QString &filePath)
-- 
2.42.0

